src/AdminConsole/Components/Pages/App/BaseApplicationPage.cs

```diff
@@ -0,0 +1,12 @@
+using Microsoft.AspNetCore.Components;
+
+namespace Passwordless.AdminConsole.Components.Pages.App;
+
+public abstract class BaseApplicationPage : ComponentBase
+{
+    /// <summary>
+    /// The route parameter for the application id for application scoped pages.
+    /// </summary>
+    [Parameter]
+    public required string App { get; set; }
```

Comment:
@jonashendrickx Just realised, this should probably be AppId for consistency sake

Reply:
@abergs No. It will not work. This binds to the 'app' route parameter. If I rename it, it will also break all the other middleware.

Reply:
Aha, I see. We should probably refactor that in a separate issue. I'll create it.